multiversx_sdk_cli/cli_shared.py

```diff
@@ -259,3 +262,23 @@ def check_if_sign_method_required(args: List[str], checked_method: str) -> bool:
             return False
 
     return True
+
+
+def prepare_forwarded_command_arguments(args: Any) -> List[str]:
+    arguments = copy.deepcopy(args)
+    args_dict: Dict[str, Any] = arguments.__dict__
+
+    # delete the function key because we don't need to pass it along
+    args_dict.pop("func")
+
+    args_list: List[str] = []
+    for key, val in args_dict.items():
+        if key == "path":
```

Comment:
This is a temporary fix, until the issue is solved in the sc-meta.

Reply:
The issue was in `mxpy` not in `sc-meta`. It's now solved, the workaround is removed.